mk/python.mk

```diff
@@ -0,0 +1,30 @@
+# check Python version
+PYTHON := $(shell which "python")
+PYTHON_COMPATIBLE_VERSION = 3.5
+
+ifndef PYTHON
```

Comment:
If `python` is not present, try `python3`. Then, if version is mismatching, try `python3.5`.